lib/cli/options.ts

```diff
@@ -42,11 +41,11 @@ export class Option {
     if (value && typeof value === 'string') {
       return '' + value;
     }
-    return null;
+    return '';
```

Comment:
Sorry, I wrote null will return type errors, I was looking at the wrong method. This does not cause type errors. I also do not think this will change any of the current behavior in the current code.